### ISSUE TITLE
Jira integration not working on any version higher than 3.15 with exact same configuration

### ISSUE THREAD

**Issue by avivek** [jenkinsci/git-changelog-plugin#72]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```
Jenkins 2.332.2
Plugin version > 3.15
Ubuntu 20 LTS OS
  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

Ubuntu 20 LTS

### Reproduction steps

With the exact same configuration jira issue attributes are successfully fetched from jira with plugin version 3.15 but on any higher configuration looks like no jira API calls are made.

The difference by looking through the commits after 3.15 in this repo is a dependency change for changelog-lib from 1.163.3 to 1.164.2

On scanning through the commits from the changelog-lib from 1.163.3 to 1.164.2 while am not fully sure there seems to be a relevant change in the below commit that could be causing the issue:
https://github.com/tomasbjerre/git-changelog-lib/commit/632fa0b341cf6e83b8b955ac989c8aba5c84dc52

Can you please check the same??

### Expected Results

Jira integration should work successfully and issue details to be fetched from Jira.

### Actual Results

No details are fetched from Jira and neither are there any errors in the console log or the jenkins master system log.

**Comment by esevland** [comment]
Same issue here. Environment:
```
Jenkins 2.332.3
Plugin version > 3.15
Ubuntu 18.4 LTS OS
```

**Comment by vtwaldo21** [comment]
OMG, thanks so much for documenting the work around to downgrade. Pulling my hair out thinking I had configured something wrong. There are 0 error messages anywhere!

To piggy back on @avivek 's sleuthing, i think the issue is simply in GitChangelogStep.java, there lacks a line for "withUseIntegrations(true)" for the jira and git code blocks. it is only in the redmine block (do people still use redmine?)

I would create a PR and fix it, but java is not my cup of tea and wouldn't know where to start.

Another workaround is to provide a garbage redmine entry, and that sets the boolean useIntegrations that is used by JIRA as well. At that point, the JIRA integration works again. I added this to my pipeline,

  redmine: [server: 'localhost'],

And version 3.24 works perfectly fine now for JIRA.
